airbyte-integrations/connectors/source-linkedin-ads/source_linkedin_ads/schemas/ad_direct_sponsored_contents.json

```diff
@@ -0,0 +1,31 @@
+{
+    "type": ["null", "object"],
```

Comment:
missing additional properties = true and json header. Please take a look in other schemas to fix that.